sample/imports/camel-case-attributes.html

```diff
@@ -0,0 +1,32 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<dom-module id="camel-case-attributes">
+  <template>
+    <camel-case-attribute fooBar="baz"></camel-case-attribute>
+  </template>
+  <script>
+  Polymer({
+    is: 'camel-case-attributes'
+  });
+  </script>
+</dom-module>
+
+<dom-module id="camel-case-attribute">
```

Comment:
is it a problem having two dom-modules in here? It is needed so we can have a valid property but reference it incorrectly. Could be put in a separate file but they kinda belong together.